pkg/pillar/evetpm/tpm.go

```diff
@@ -377,7 +380,7 @@ func FetchVaultKey(log *base.LogObject) ([]byte, error) {
 	//First try to read from TPM, if it was stored earlier
 	key, err := readDiskKey()
 	if err != nil {
-		log.Noticef("Generating VaultKey")
+		log.Noticef("can't read the legacy disk key, generating a new one")
```

Comment:
Why are you calling this "legacy" in the updated log messages?

Reply:
`writeDiskKey()/readDiskKey()` are used to store and read a key to/from TPM NV when SHA256 banks are not available (so no PCR policy or anything attached to the NV, just a key stored out of the disk), and it is called legacy in other parts of the code, e.g. : 
```
func isLegacyKeyPresent() bool {
	_, err := readDiskKey()
	return err == nil
}
```
I can rename it if you think it is misleading.

Reply:
Thats ok. I didn't see that context in the diffs.